providers/base/units/audio/jobs.pxu

```diff
@@ -698,13 +700,14 @@ flags: also-after-suspend
 imports: from com.canonical.plainbox import manifest
 requires:
   manifest.has_audio_playback == 'True'
+  audio_card.playback == 'supported'
```

Comment:
Hi, I followed the discussion of this PR a few weeks ago, so I just took a look and am a little bit curious about this `requires: audio_card.playback == 'supported'`.

If I understand correctly, considering the following situation,
When `manifest` is set to `True` (the device should have an audio playback feature), but the resource job `audio_card` **cannot** detect the playback devices so is NOT `supported`. This `audio/detect-playback-devices` job should fail. However, adding this `requires: audio_card.playback == 'supported'` will make this job be skipped. 

Same as the `audio/detect-capture-devices` below.